### ISSUE TITLE
Consider archiving this repo

### ISSUE THREAD

**Issue by JackWilb** [multinet-app/multinet-deploy#51]
Since our deployment strategy has changed and this repo is unused, we should consider archiving it. I'm opening this issue as a place to discuss this proposal and I'd love to hear your thoughts.

**Comment by waxlamp** [comment]
Agreed that this should be archived. I'll take care of that now. If there are objections, we can always unarchive it.
